main.go

```diff
@@ -40,8 +40,12 @@ func createGitHubClient(tokenEnv, endpointEnv string) (github.Client, error) {
 	if endpoint == "" {
 		endpoint = "https://api.github.com"
 	}
+	proxy := os.Getenv(proxyEnv)
+	if proxyEnv == "" {
+		proxy = ""
+	}
```

Comment:
This is not necessary because `proxyEnv` is either `GITHUB_MIGRATOR_SOURCE_PROXY_URL` or `GITHUB_MIGRATOR_TARGET_PROXY_URL`.
```suggestion
```